Makefile

```diff
@@ -1,22 +1,52 @@
+RHEL5=$(shell grep -q -i 'release 5' /etc/redhat-release && echo 1)
+F27=$(shell grep -q -i 'release 27' /etc/redhat-release && echo 1)
 USE_SELINUX=$(shell test -d /sys/fs/selinux && echo ":Z")
 DOCKERFILE  ?= $(CURDIR)/images/Dockerfile.el7
+ifeq ($(F27), 1) # no pip for F27, only pip3
+PYTHON ?= python3
+PIP ?= pip3
+else
 PYTHON ?= python2
+PIP ?= pip
+endif
+
+default: help
+
+help:
+	@echo "Usage: make [target] <ARG=value, ...>"
+	@echo
+	@echo "target:"
+	@echo "  help          show this message"
+	@echo "  install       install locally with default python"
+	@echo "  test-python3  test locally with python 3"
+	@echo "  test-python2  test locally with python 2"
+	@echo "  docker-build  build the docker image"
+	@echo "  docker-run    run bash in a preconfigured docker container"
+	@echo "  docker-test   test in a docker container"
+	@echo
+	@echo "python args:"
+	@echo "  PYTHON        python executable to use (python2, python3)"
+	@echo
+	@echo "docker-* args:"
+	@echo "  DOCKERFILE    dockerfile to use (one of images/Dockerfile.*)"
 
 install:
 	cd src/ && $(PYTHON) setup.py install
+ifneq ($(RHEL5), 1) # no pip for RHEL5
```

Comment:
The install target installs the tools into python's site-packages and I don't think we should do anything with the test requirements here

Reply:
Ah, I see. What about a `test-install` target?